ui-core/src/styles/inputs/inputWithSuggestions.css

```
@@ -0,0 +1,45 @@
+.input-with-suggestions {
+  position: relative;
+  width: 100%;
+  @apply text-grey-80;
+  font-size: 1rem;
+
+  .suggestions-list {
+    position: absolute;
+    width: calc(100% - 2.1rem);
+    top: calc(100% - 1.4rem);
+    left: 1rem;
+    right: 0;
+    z-index: 1000;
+    list-style: none;
+    max-height: calc(2.5rem * var(--number-of-suggestions));
+    overflow-y: auto;
+    @apply bg-white-25;
+    border-radius: 0.375rem;
+    box-shadow:
+      0px 6px 21px -5px rgba(24, 68, 239, 0.26),
+      0px 8px 30px -5px rgba(0, 0, 0, 0.16),
+      0px 3px 5px -2px rgba(0, 0, 0, 0.1);
+  }
+
+  .suggestion-item {
+    padding: 0.688rem 0.688rem 0.813rem 1.188rem;
```

I think we can assume that padding horizontal will be the same on both sides

```suggestion
    padding: 0.688rem 1.188rem 0.813rem;
```

---

Fixed